packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/Messages.md

```diff
@@ -271,6 +287,35 @@ You can add custom content to specific parts of a `<Message>` via the `extraCont
 
 ```
 
+### Custom message structure
+
+For more advanced use cases, you can build completely custom message structures by passing children directly to `<Message>`. This approach is useful when you need to customize the order or structure of message elements beyond what the standard props allow.
+
+When creating custom message structures, you must follow an intended composable structure:
+
+1. **Message content and actions** must be wrapped in `<MessageAndActions>`. This includes, but is not limited to:
+
+   - `<MarkdownContent>` - For rendering markdown or plain text content
```

Comment:
```suggestion
When creating custom message structures, you must follow an intended composable structure.

1. **Message content and actions:** Wrap in `<MessageAndActions>`. This includes, but is not limited to:

   - `<MarkdownContent>`: For rendering markdown or plain text content
```
formatting changes - might look a little funky, but is technically fairly consistent with other docs